source/runbooks/revoke-network-access.html.md.erb

```diff
@@ -0,0 +1,96 @@
+---
+owner_slack: "#modernisation-platform"
+title: Revoke network access if required
+last_reviewed_on: 2024-03-04
+review_in: 6 months or when is resolved.
+---
+
+# <%= current_page.data.title %>
+
+# Summary
+
+The steps listed deny access on a platform level. If additional changes are required at an application level then these must be undertaken by the application owning team.
+
+A network attack is an exceptional situation so quick amendments are needed. It is anticiapted that this can be done quickley through the console.
+
+This document lists the steps needed to stop access to and from the internet and prevening the system from brineg accessed from or accessing other applications on the MoJ.
+
+All changes are to be made manually on the relevant environment (for example core-vpc-production) and are not made through the code. If a way of reverting back to baseline is required the terraform code and be run and this will reset everything.
+
```

Comment:
At this point it would be good to mention (perhaps in a warning NOTE format) all the risks that the manual changes carry, see the [slack thread ](https://mojdt.slack.com/archives/C013RM6MFFW/p1709552392385159)for more information

Reply:
Yes. I missed this

Reply:
Mistakenly added to the previous thread
https://github.com/ministryofjustice/modernisation-platform/pull/6380#discussion_r1511242225

Reply:
Your warning is regarding the changes impact (which is already covered in this documentation), but I am looking for a warning about the risks that manual changes (vs code changes) carry, since we are doing it through click-ops and not through code.

Reply:
Can you clarify this please?

Reply:
What I was after and what was already discussed in the [slack thread](https://mojdt.slack.com/archives/C013RM6MFFW/p1709559726905539?thread_ts=1709552392.385159&cid=C013RM6MFFW) is something along the lines of: 
These manual changes mean that:
- there is no history of changes in code and therefore if there are additional changes done that are not part of current IaC (e.g. introducing a new route table), these will need noting
- there is a danger of unintentionally undoing these changes with a pipeline run, hence for the time when the manual changes are in place, no pipeline relevant to the networking bits that were updated can be run

Reply:
So for the avoidance of doubt @SteveLinden , this could be met by adding a [blockquote](https://www.markdownguide.org/basic-syntax/#blockquotes-1) in near the top of the document that says those things.

Reply:
Additional comment added